src/adapters/peggedAssets/usual-usd/index.ts

```diff
@@ -0,0 +1,8 @@
+const chainContracts = {
+    ethereum: {
+      issued: ["0x73A15FeD60Bf67631dC6cd7Bc5B6e8da8190aCF5"],
```

Comment:
are all the tokens existing tokens backed and issued? or do you perhaps keep a buffer of unissued tokens somewhere that you then use to distribute tokens?

i looked at etherscan holders and spotted some address that could fit that bill but wanted to verify if they were normal holders or not